src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -305,6 +304,7 @@ def assemble_json(ids):
         if full_id_list:
             setattr(namespace, '_ids', full_id_list)
 
+        from msrestazure.tools import parse_resource_id, is_valid_resource_id
```

Comment:
You should move to azure-mgmt-core, since it's where future update will be. This is the same code I copy pasted without any changes:
```python
from azure.mgmt.core.tools import  parse_resource_id, is_valid_resource_id
```

Reply:
+@jsntcy Could you help address this, and do necessary tests in a separate PR?

Reply:
Did the change in this PR already.